tests/yast2_gui/yast2_bootloader.pm

```diff
@@ -55,7 +55,9 @@ sub run {
     #	proctect boot loader with password
     assert_and_click 'yast2-bootloader_protect-bootloader-with-password';
     send_key 'alt-p';
+    wait_still_screen 3;
```

Comment:
The very basic problem of ```wait_still_screen``` is that it returns a value which denotes whether the screen was still or not, but we do not work the return code at all. So, what happens over here is, that we are checking if the screen is going to be still for 3 seconds, within 30 seconds timeout, while the similarity level is set to default value https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L631. I am not sure what kind of return codes you get, but there might be a potential problem on machines other than x86_64.

You might use [``type_string_slow_extended``](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/utils.pm#L326) which exposes ``wait_still_screen`` arguments in ``type_string`` calls.

Usage: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/yast2_gui/yast2_hostnames.pm

Reply:
@mloviska could you please check `type_string_slow_extended` call at lines 59 and 62 if it is correct. And i'm not sure if and how to change `wait_still_screen` at line 49.

Reply:
LGTM, it might happen that you need to change the value of some argument like *similarity_level*.  Please use type_string directly in such a case.